### ISSUE TITLE
GitLab Pull Request Support

### ISSUE THREAD

**Issue by Azzurrio** [drone/drone#459]
I did setup drone on Amazon ec2 instance and I did configure drone with gitlab. Actually drone now run merges with upstream repo successfully, but the problem comes with pull requests. Once anyone submit a pull request to the upstream repo, I got the following error:
```
$ git clone --depth=50 --recursive --branch=master git@git.whitespace.company:whitespace/incident_report.git /var/cache/drone/src/git.whitespace.company/whitespace/incident_report
Cloning into '/var/cache/drone/src/git.whitespace.company/whitespace/incident_report'...
Warning: Permanently added 'git.whitespace.company,54.186.255.10' (ECDSA) to the list of known hosts.
$ git fetch origin +refs/pull/32/head:refs/remotes/origin/pr/32
fatal: Couldn't find remote ref refs/pull/32/head
```
So, any suggestions what's the problem here?

**Comment by partikus** [comment]
hello, @bradrydzewski Does drone.io support gitlab's merge request hooks? I've installed drone.io locally and it looks like the build for MR is not running

[here](https://github.com/drone/drone/blob/master/plugin/remote/gitlab/gitlab.go#L240) it looks that drone.io does not support MR yet

**Comment by Bugagazavr** [comment]
currently - no, since `7.14` release i have plan to implement this in `0.4` branch

**Comment by kpacha** [comment]
I created a PR to send back to gitlab the status of a build as a comment on the related issue. You can check it here: #1141 

I hope it will do the work until the release of 0.4

**Comment by bradrydzewski** [comment]
this was merged in 0.4 and can be closed now
